### PR TITLE
[TCBZ2616] Update to latest extension version series

### PR DESCRIPTION
**Pull request by spbrogan**

### CODE REVIEW

**Comment by spbrogan** [comment]
Link to bugzilla
https://bugzilla.tianocore.org/show_bug.cgi?id=2616

File: pip-requirements.txt [hunk 1]
```diff
@@ -9,8 +9,10 @@
 # https://pypi.org/project/pip/
 # https://pip.pypa.io/en/stable/user_guide/#requirements-files
 # https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format
-#
+# https://www.python.org/dev/peps/pep-0440/#version-specifiers
 ##
 
 edk2-pytool-library==0.10.*
```
Comment (corthon) [inline pip-requirements.txt:15]: Do we want to do the same thing on Library while we're at it, to make sure that Library isn't too far off of the new Extensions base version?

**Review by corthon — CHANGES_REQUESTED** [review]
Needs MU_CHANGE

File: pip-requirements.txt (hunk 1 above)
Comment (spbrogan) [reply pip-requirements.txt:15]: I think i am ok leaving it free floating within the 0.10.x series.  10.x is our latest version  All docs say you should run pip install --upgrade and thus it should get upgraded.

**Pull request merged**